grails-app/conf/plugin.groovy

```diff
@@ -29,9 +29,11 @@
 auth.admin_role = "ROLE_ADMIN"
 serverName = 'http://dev.ala.org.au:8080'
 
-dataquality.enabled = false
-dataquality.baseUrl = 'https://biocache.ala.org.au/data-quality/'
+dataquality.enabled = true
```

Comment:
Not sure it should ON be default - most LA deployments will not use it.

Reply:
I'll rever it then.